### PR TITLE
Created features to add, edit and remove tasks

### PR DESCRIPTION
**Pull request by alonsofl**
### Summary
Added features using plain JS to add, edit and remove tasks.
>Please note that to use the remove function (the trash icon to appear) you should click on the input. Just like the example for the exercise.

#### Details
- Added script to add tasks from the input.
- Added script to remove items by clicking on the trash icon that appears when clicking on the input.
- Added script to clear completed tasks.

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @alonsofl,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Add More Info about how to download the repo locally and how to make it runnig.
- Add function to the remove or trash icon trash

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -19,10 +18,6 @@
 
 ![Screenshot](./screenshot.png)
 
-### Links
-
-- [Live Site](https://alonsofl.github.io/todo-list/)
-
 ## My process
 
 ### Built with
```
Comment (dasileker) [inline README.md:23]: - [ ] Please add how to make the project running on a local machine.

File: src/modules/task-util.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    if (!parentElement.children[3]) {
+      parentElement.appendChild(removeIcon);
+    }
+
+    removeIcon.addEventListener('click', (e) => {
+      removeSelectedItem(toDoList, e);
+    });
+  }
+};
```
Comment (dasileker) [inline src/modules/task-util.js:71]: - [ ] Please add the trash or remove icon, to remove only the targeted array in your to-do list.

Comment (alonsofl) [reply src/modules/task-util.js:71]: Hello @dasileker it is on `task-util.js` removeSelectedItem. And it's working fine.

Comment (dasileker) [reply src/modules/task-util.js:71]: sorry, my friend but you need a  beside the drag dots so we can remove only the selected one

Comment (alonsofl) [reply src/modules/task-util.js:71]: @dasileker have you saw the project requirements?
https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/

File: README.md (hunk 1 above)
Comment (DeliceLydia) [reply README.md]: Resolved this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @alonsofl , 

Your project is complete! You did an amazing job on this project and I commend you for your communication with the previous reviewer.. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations once again! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**